manifests/params.pp

```diff
@@ -225,4 +225,7 @@
       $plugin_prefix = 'smart_proxy_'
     }
   }
+
+  $puppet_use_cache = undef
+  $puppet_cache_location = '/var/lib/smart-proxy/cache'
```

Comment:
Changed and let puppet create the directory so we don't have to wait for new packages.